config/examples/backend.yml

```diff
@@ -1,18 +1,25 @@
+base: &default
+  url: <%= ENV.fetch('BACKEND_URL', 'http://localhost:3001') %>
+  public_url: <%= ENV.fetch('BACKEND_PUBLIC_URL') { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
+  persistent: <%= ENV.fetch('BACKEND_PERSISTENT', '1') == '1' %>
+  max_tries: <%= ENV.fetch('BACKEND_MAX_TRIES', 5) %>
+  open_timeout: <%= ENV.fetch('BACKEND_OPEN_TIMEOUT', 3) %>
+  timeout: <%= ENV.fetch('BACKEND_TIMEOUT', 3) %>
+
+  # superseded by 'url'
+  host: <%= ENV.fetch('BACKEND_HOST', 'localhost:3001') %>
```

Comment:
url takes precedence https://github.com/3scale/porta/blob/c499550/app/lib/backend_client.rb#L23

host+secure could potentially be removed, but host is actually used here: https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/app/views/api/integrations/plugin/plugins/_rest.html.erb#L5

But this code is not visible anymore, and I think we can even review and remove it.